debugging/src/buggy.cpp

```diff
@@ -21,14 +21,13 @@ class Student {
 
 Student* getStudent(){
   Student* myStudent = new Student(10801983,"Mary Wright");
```

Comment:
given this `new` statement and seeing no delete anywhere this code has a memory leak.